### PR TITLE
D4N: Make shutdown method specific to LFUDA

### PR DESCRIPTION
**Pull request by samarahu**

### CODE REVIEW

File: src/rgw/driver/d4n/d4n_policy.h [hunk 1]
```diff
@@ -121,6 +119,7 @@ class LFUDAPolicy : public CachePolicy {
         return nullptr;
       return it->second;
     }
+    void shutdown();
```
Comment (pritha-srivastava) [inline src/rgw/driver/d4n/d4n_policy.h:122]: Method definition?

Comment (samarahu) [reply src/rgw/driver/d4n/d4n_policy.h:122]: Are you asking for a description of what it does?

Comment (pritha-srivastava) [reply src/rgw/driver/d4n/d4n_policy.h:122]: No, I am asking for a definition of this method, I believe its already there?

Comment (samarahu) [reply src/rgw/driver/d4n/d4n_policy.h:122]: Yes, it's in the .cc file

**Pull request closed without merging**